src/python/pants/base/exception_sink.py

```diff
@@ -59,14 +59,17 @@ def _toggle_ignoring_sigint(self, toggle: bool) -> None:
             with self._ignore_sigint_lock:
                 self._ignoring_sigint = toggle
 
-    def handle_sigint(self, signum: int, _frame):
+    def _send_signal_to_children(self, _received_signal: int) -> None:
         self_process = psutil.Process()
         children = self_process.children()
-        logger.debug(f"Sending SIGINT to child processes: {children}")
+        logger.warning(f"Sending SIGINT to child processes: {children}")
```

Comment:
It would be good to replace SIGINT with the _received_signal or signal.strsignal(_received_signal) or signal.getsignal(_received_signal).

Reply:
`signal.strsignal` apparently only exists after python 3.8